package.json

```diff
@@ -8,7 +8,9 @@
   "scripts": {
     "lint": "eslint .",
     "pretest": "node test/helpers/pretest.js",
-    "test": "grunt",
```

Comment:
agreed, discussed with Simon, the current integration tests are actually end to end tests and will be moved to acceptance tests in the next PR. I will then add unit and integration tests.